src/main/resources/application.conf

```diff
@@ -44,6 +44,27 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+
+  // For any backend that assumes a local filesystem (local, sge)
+  local-filesystem {
+    // Root directory where Cromwell writes job results to.  This directory must be
+    // visible and writeable to the Cromwell process as well as the jobs that Cromwell
+    // launches
+    root: "cromwell-executions"
+
+    // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
+    // The following are strategies used to make those links.  They are ordered.  If one fails
+    // The next one is tried:
+    //
+    // hard-link: attempt to create a hard-link to the file
+    // copy: copy the file
+    // soft-link: create a symbolic link to the file
+    //
+    // NOTE: when using Docker, do NOT use soft-link
```

Comment:
This seems kinda dangerous since "application.conf" is write-once and "docker?" is a WDL choice... i.e. If this .conf is on a server and someone submits a workflow with a docker specified, does everything explode?